test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Query/Expressions/FilterBinderTests.cs

```diff
@@ -2363,10 +2363,10 @@ public void Isof_WithNullTypeName_ThrowsArgumentNullException(string filter)
             // Arrange & Act & Assert
 #if NETCOREAPP3_1
             ExceptionAssert.Throws<ArgumentNullException>(() => Bind<Product>(filter),
-                "Value cannot be null. (Parameter 'qualifiedName')");
+                "Value cannot be null. (Parameter 'typeName')");
```

Comment:
Why this change?

Reply:
This is caused by changes that were made in ODL..This PR: https://github.com/OData/odata.net/issues/2607